### PR TITLE
88_Tracy Salander 第一次作业

### PR DESCRIPTION
**Pull request by TracySalander**
comment for the solution
@topmorphs

### CODE REVIEW

File: Lesson-1/assignment/yours.sol [hunk 1]
```diff
@@ -1 +1,97 @@
 /*作业请提交在这个目录下*/
+Assignment1: make a change to fit general employee;
+The original version:
+pragma solidity ^0.4.14;	// tell compiler the version of program which is good for 
+							//outsourcing interaction 
+
+contract Payroll{	// like Class but contract
+	uint salary = 1 ether;	// Tracy's salary
+	address Tracy = dsfasdfs235r1345; // the wallet address of Tracy
```
Comment (topmorphs) [inline Lesson-1/assignment/yours.sol:9]: use tracy instead "Tracy"

File: Lesson-1/assignment/yours.sol [hunk 2]
```diff
@@ -1 +1,97 @@
...
+						// meaning it will refund the remaining gas. 
+		}
+		lastPayday = nextPayday;	// here we don't use now() because Tracy can get several
+									// months' salary
+		Tracy.transfer(salary;)
+	}
+}
+
+Question: can we modify the code to let it suits to other person and change the salary?
+
```
Comment (topmorphs) [inline Lesson-1/assignment/yours.sol:47]: yes. You will learn it in the next lesson

**Review by topmorphs — APPROVED** [review]

**Comment by TracySalander** [comment]
@topmorphs

**Review by topmorphs — APPROVED** [review]
Great work!

**Comment by TracySalander** [comment]
@topmorphs

**Pull request merged**